docs/_pages/en/dsra_scenario_map.md

```diff
@@ -153,6 +154,11 @@ crossorigin=""></script>
         sauidLayer.setFeatureStyle( selection, selectedStyle(), 100 );
       });
 
+      // Add a popup with desired property
+      L.popup().setContent( "<strong>Personnes déplacées après 90 jours: </strong>" + e.layer.properties.sCt_Res90_b0.toString() )
```

Comment:
Is this supposed to be in English?

Reply:
Fixed. I am apparently very sloppy today. Thanks for catching that.